UI/volume-control.cpp

```diff
@@ -157,8 +197,28 @@ VolControl::VolControl(OBSSource source_, bool showConfig)
 	mute->setAccessibleName(
 			QTStr("VolControl.Mute").arg(sourceName));
 
+#if defined(_WIN32) || defined(__APPLE__) || HAVE_PULSEAUDIO
+	headphone->setAccessibleName(
+			QTStr("VolControl.Headphones").arg(sourceName));
+	headphone->setToolTip(
+			QTStr("VolControl.Headphones").arg(sourceName));
+
+	switch (obs_source_get_monitoring_type(source)) {
+		case OBS_MONITORING_TYPE_NONE:
```

Comment:
Style:  Don't add extra indentation to `case` statements.

Reply:
It is now clang formatted.